ParseTree/ParseTreeTests/ParseTreeTests.cs

```diff
@@ -0,0 +1,77 @@
+using ParseTree;
+namespace ParseTreeTests;
+
+public class ParseTreeTests
+{
+    private ParseTree.ParseTree parseTree = new();
+
+    [SetUp]
+    public void Setup()
+    {
+        parseTree = new();
+    }
+
+    [TestCase("(* (+ 1 1) (* 5 (/ -10 8)))", ExpectedResult = -12.5)]
+    [TestCase("(/ (+ (* 13 3) 1) (* 2 (/ (- 20 8) 6)))", ExpectedResult = 10)]
+    [TestCase("/ 0 -5", ExpectedResult = 0)]
+    public double BuildTreeAndCalculateTree_WithCorrectExpression_ShouldReturnExpectedResult(string expression)
+    {
+        parseTree.BuildTree(expression);
+        return parseTree.CalculateTree();
+    }
+
+    [TestCase("(* (+ 1 1) (* 5 (/ -10 8)))")]
+    [TestCase("(/ (+ (* 13 3) 1) (* 2 (/ (- 20 8) 6)))")]
+    [TestCase("(/ 0 -5)")]
+    public void BuildTreeAndPrint_WithCorrectExpression_ShouldReturnExpectedResult(string expression)

```

Comment:
Тут кажется, что тестируется именно `Print`, BuildTree уже протестировано в прошлом тесте (причем с такими же данными). Пока искал тест `Print` после BuildTree перестал читать :)

```suggestion
    public void Print_WithCorrectExpression_ShouldReturnExpectedResult(string expression)
```